pkg/virtualKubelet/apiReflection/reflectors/incoming/replicaSets.go

```diff
@@ -83,7 +83,7 @@ func (r *ReplicaSetsIncomingReflector) preDelete(obj interface{}) interface{} {
 		return nil
 	}
 
-	po, err := r.GetCacheManager().GetHomeNamespacedObject(apimgmt.Pods, homeNamespace, foreignReplicaSet.Name)
+	po, err := r.GetCacheManager().GetHomeNamespacedObject(apimgmt.Pods, homeNamespace, podName)
```

Comment:
@mlavacca In order to use a unique mock cache, I had to change a little the maps in the mock struct. After that, the tests on this function were no more passing. I found that probably this was wrong, can you confirm it?

PS: if you confirm I will squash these commits